tests/Spout/Reader/CSV/ReaderTest.php

```diff
@@ -384,7 +393,7 @@ private function getAllRowsForFile(
     public function dataProviderForTestReadCustomEOL()
     {
         return [
-            ['csv_with_CR_EOL.csv', "\r"],
+//            ['csv_with_CR_EOL.csv', "\r"],
```

Comment:
My bad, it does not. You need to add this code in CSV/Reader::openReader():

```php
$this->autoDetectLineEndings = ini_get('auto_detect_line_endings');
ini_set('auto_detect_line_endings', true);
```

and reset the value in the CSV/Reader::closeReader() function:

```php
ini_set('auto_detect_line_endings', $this->autoDetectLineEndings);
```